docs/serverless/cloud-native-security/falco-setup.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[ingest-falco]]
+= Ingest CNCF Falco data
+
+CNCF Falco is an open-source runtime security tool that detects anomalous activity in Linux hosts, containers, Kubernetes, and cloud environments. You can ingest Falco alerts into {es} to view them on {elastic-sec}'s Alerts page and incorporate them into your security workflows by using Falcosidekick, a proxy forwarder which can send alerts from your Falco deployments to {es}.
+
+First, you'll need to configure {elastic-sec} to receive data from Falco, then you'll need to configure Falco and Falcosidekick to send data to {es}. 
+
+[discrete]
+[[ingest-falco-setup-kibana]]
+== Configure {elastic-sec} to receive Falco data
+
+In {elastic-sec}:
+
+. Click **Add integrations**.
+. Search the Integrations page for `Falco`, then select it.
+. Go to the Falco integration's **Settings** tab. 
+. Click **Install Falco**, then confirm by clicking **Install Falco** again. Installation should take less than a minute. 
+
+{elastic-sec} is now ready to receive data from Falco. The Falco integration page now has an **Assets** tab where you can inspect the newly installed assets that help to ingest Falco data.
+
+Next, to make alerts from Falco appear on {elastic-sec}'s Alerts page:
+
+. Find the **Rules** page in the navigation menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

Comment:
The following edit is a bit more straightforward and removes the need for step 2.

```suggestion
. Find the **Detection rules (SIEM)** page in the navigation menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```